woocommerce/woocommerce-ios
# REST API: Update an icon on Jetpack benefit modal

Closes: #8912 


## Description

This PR updates the icon for the multiple stores benefit on the Jetpack benefit modal.

## Testing instructions

- Log out of the app or skip onboarding if needed.
- Log in to a self-hosted site without Jetpack.
- On the dashboard screen, tap the Jetpack benefit banner or select Install Jetpack from the Settings screen.
- Notice that the Jetpack benefit modal displays the correct icon for the multiple stores benefit

## Screenshots

<img src="https://user-images.githubusercontent.com/5533851/224660715-6ceb2e5f-4526-40d5-be27-896ce773b30a.png" width=320 />


---
